### PR TITLE
sc-56: Sc 56/test script for contracts

### PR DESCRIPTION
**Pull request by akshay-rakheja**
Sample test script for contracts. Need to edit as per methods in market.sol.

### CODE REVIEW

**Review by Justin1002 — APPROVED** [review]

File: test/sample-test.js [hunk 1]
```diff
@@ -1,19 +1,42 @@
...
+        price: i.price.toString(),
+        tokenId: i.tokenId.toString(),
+        seller: i.seller,
+        owner: i.owner,
+        tokenUri
+      }
+      return item
+    }))
+    console.log('items: ', items)
+  })
```
Comment (connie-ho) [inline test/sample-test.js:41]: maybe im missing something but should there be an expect statement for tis test?

Comment (akshay-rakheja) [reply test/sample-test.js:41]: Not sure yet but I think it's not required here

Comment (connie-ho) [inline test/sample-test.js:41]:
```suggestion
  })
  
  expect(items).to.deep.equal([{
    price: 1,
    //... other fields
  }, {
    price: 2,
    //... other fields in the item object to assert
  }])
```

**Review by connie-ho — CHANGES_REQUESTED** [review]

**Comment by connie-ho** [comment]
merge conflicts ):

**Pull request closed without merging**